examples/extended/parallel/MPI/source/src/G4MPImanager.cc

```diff
@@ -173,14 +181,14 @@ void G4MPImanager::Initialize()
   g4mpi_ = this;
 
   // get rank information
-  world_size_ = MPI::COMM_WORLD.Get_size();
+  MPI_Comm_size(MPI_COMM_WORLD, &world_size_);//world_size_ = MPI::COMM_WORLD.Get_size();
```

Comment:
```suggestion
  MPI_Comm_size(MPI_COMM_WORLD, &world_size_);
```

Remove trailing comment